src/test/scala/mesosphere/marathon/api/v2/TasksResourceTest.scala

```diff
@@ -77,7 +77,7 @@ class TasksResourceTest extends UnitTest with GroupCreation {
       Given("no apps")
       config.zkTimeoutDuration returns 5.seconds
       taskTracker.instancesBySpec returns Future.successful(InstanceTracker.InstancesBySpec.empty)
-      groupManager.rootGroup() returns createRootGroup()
```

Comment:
Why is this line deleted? Shouldn't the following one be just added?

Reply:
I have the same question.

Reply:
indexJson method does not call rootGroup(), just indexTxt does so I think it was a dead line of code even before I started touching this so I just removed it...